docs/Readme.md

```diff
@@ -14,6 +14,9 @@ If you would like to see more examples of field data plug-ins, please visit our
 
 ## Change Log
 
+### AQTS 2018.4
+- Changed format of ControlConditionActivity.ControlCondition from Enum to string. New string value contains identifier for control condition picklist item. 
```

Comment:
This isn't accurate as the type is will be a `ControlConditionPickList` not a `string`.

This will need to be rebased when PR #34  is merged but this could be more consistent with that change by explicitly marking this as a breaking change ex:

- **Breaking change**: Changed the parameter type of ControlCondition.ControlCondition from the ConditionType Enum to ControlConditionPickList. See [PickList Data Type](https://github.com/AquaticInformatics/aquarius-field-data-framework/blob/master/docs/Readme.md#picklist-data-type) for more details about working with PickLists.